fetch.js

```diff
@@ -459,6 +463,10 @@
         xhr.setRequestHeader(name, value)
       })
 
+      if (init.signal && init.signal.addEventListener) {
+        init.signal.addEventListener('abort', xhr.abort)
```

Comment:
Should we remove the event listener also in the `xhr.onabort` function?

Reply:
Yup. I'll do that.